test/unit/components/VPaper/VPaper.spec.js

```diff
@@ -0,0 +1,35 @@
+import { test } from '@/test'
+import VPaper from '@/components/VPaper'
+
+test('VPaper.vue', ({ mount }) => {
+  it('should change elevation on mouseover', async () => {
+    const wrapper = mount(VPaper, {
+      propsData: {
+        hover: 5
+      }
+    })
+
+    expect(wrapper.vm.$el.classList.contains('elevation-0')).toBe(true)
+
+    const listeners = wrapper.vm.$options.computed.listeners
```

Comment:
i'd rather test here the bevaviour instead of internal implementation (trigger the proper event on `vm.$el` and check `vm.$el.classList`) - for both versions (with and without `hover` prop)

Reply:
No longer necessary, moved functionality to v-card